File: src/main/java/com/my/admin/app/commerce/model/api/response/TurnResponse.java
package com.my.admin.app.commerce.model.api.response;

import com.fasterxml.jackson.annotation.JsonInclude;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

@Data
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class TurnResponse {

    @ApiModelProperty(value = "Turn Id", example = "1")
    private String id;

    @ApiModelProperty(value = "Turn name", example = "Tarde")
    private String name;
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/model/api/response/TurnResponse.kt
package com.bk.reactive.app.commerce.admin.my.model.api.response

import com.fasterxml.jackson.annotation.JsonInclude
import io.swagger.annotations.ApiModelProperty

@JsonInclude(JsonInclude.Include.NON_EMPTY)
class TurnResponse(
    @ApiModelProperty(value = "Turn Id", example = "1")
    val id: String,

    @ApiModelProperty(value = "Turn name", example = "Tarde")
    val name: String
)